src/constants.ts

```diff
@@ -13,7 +18,8 @@ export const Urls = {
     PERSONAL_CODE: `${urlWithTransactionIdAndSubmissionId}/individual/personal-code`,
     INDIVIDUAL_STATEMENT: `${urlWithTransactionIdAndSubmissionId}/individual/psc-statement`,
     PSC_VERIFIED: `${urlWithTransactionIdAndSubmissionId}/psc-verified`,
-    RLE_LIST: `${urlWithTransactionIdAndSubmissionId}/rle/rle-list`
+    RLE_LIST: `${urlWithTransactionIdAndSubmissionId}/rle/rle-list`,
+    STOP_SCREEN: `${urlWithTransactionIdAndSubmissionId}/stop/:stopType`
```

Comment:
The Secure Company Stop screen I'm working on, comes before the Trans and Submission gets generated, so doesn't have this in the URL path.  Can we handle that??

Reply:
Good spot, i'm making changes to allow for stop screens without those IDs in the URL

Reply:
👍